apps/course/src/app/shared/navigation/navbar-course/navbar-course.component.html

```diff
@@ -40,7 +40,7 @@
         </ul>
       }
     </div>
-    <a class="btn btn-ghost text-xl" routerLink="/">
+    <a class="btn btn-ghost text-xl" href="../c">
```

Comment:
This isn't dynamic and will throw routing error sooner or later. Also this still isn't what I expected.